clients/java/src/test/java/io/camunda/client/tenant/SearchClientByTenantTest.java

```diff
@@ -56,7 +56,7 @@ void shouldIncludeSortAndPaginationInSearchRequestBody() {
     final String requestBody = lastRequest.getBodyAsString();
 
     assertThat(requestBody).contains("\"sort\":[{\"field\":\"clientId\",\"order\":\"DESC\"}]");
-    assertThat(requestBody).contains("\"page\":{\"limit\":2,\"after\":null,\"before\":null}");
+    assertThat(requestBody).contains("\"page\":{\"limit\":2}");
```

Comment:
ℹ️ @nicpuppa and @megglos, this is a result of the change to the mustache template mentioned above.